jekyll/_cci2/server/installation/installation-reference.adoc

```diff
@@ -252,145 +256,157 @@ a|
 |bool
 |`false`
 |
 
-|global.tracing.sample_rate
+|`global.tracing.sample_rate`
 |float
 |`1`
 |
 
-|insights_service.dailyCronHour
+|`insights_service.dailyCronHour`
 |int
 |`3`
 |Defaults to 3AM local server time.
 
-|insights_service.hourlyCronMinute
+|`insights_service.hourlyCronMinute`
 |int
 |`35`
 |Defaults to 35 minutes past the hour.
 
-|insights_service.isEnabled
+|`insights_service.isEnabled`
 |bool
 |`true`
 |Whether or not to enable the insights-service deployment.
 
-|insights_service.replicas
+|`insights_service.replicas`
 |int
 |`1`
 |Number of replicas to deploy for the insights-service deployment.
 
-|internal_zone
+|`internal_zone`
 |string
 |`"server.circleci.internal"`
 |
 
-|keyset
+|`keyset`
 |object
 |`{"encryption":"","signing":""}`
-|Keysets (2 Options) used to encrypt and sign artifacts generated by CircleCI. You need these values to configure server. Option 1: Set the values keyset.signing and keyset.encryption here and CircleCI will create the secret automatically. Option 2: Leave this blank, and create the secret yourself. CircleCI will assume it exists. The secret must be named 'signing-keys' and have the keys; signing-key, encryption-key.
+a| Keysets used to encrypt and sign artifacts generated by CircleCI. You need these values to configure server.
+
+**Option 1:** Set the values `keyset.signing` and `keyset.encryption` here and CircleCI will create the Kubernetes Secret automatically. 
+
+**Option 2:** Leave this blank, and create the secret yourself. CircleCI will assume it exists. The secret must be named 'signing-keys' and have the keys: `signing-key`, `encryption-key`.
 
-|keyset.encryption
+|`keyset.encryption`
 |string
 |`""`
-|Encryption Key To generate an artifact ENCRYPTION key run: `docker run circleci/server-keysets:latest generate encryption -a stdout`
+|Encryption key to generate an artifact ENCRYPTION key run: `docker run circleci/server-keysets:latest generate encryption -a stdout`
 
-|keyset.signing
+|`keyset.signing`
 |string
 |`""`
 |Signing Key To generate an artifact SIGNING key run: `docker run circleci/server-keysets:latest generate signing -a stdout`
 
-|kong.acme.email
+|`kong.acme.email`
 |string
 |`"your-email@example.com"`
 |
 
-|kong.acme.enabled
+|`kong.acme.enabled`
 |bool
 |`false`
-|This setting will fetch and renew Let's Encrypt certs for you. It defaults to false as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run helm upgrade with the updated setting if you want.
+|This setting will fetch and renew Let's Encrypt certs for you. It defaults to `false` as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run `helm upgrade` with the updated setting if you want.
```

Comment:
Should we link out to Let's Encrypt here? Unless it's linked out to elsewhere on the page, I'd suggest it.